src/sentry/static/sentry/app/components/acl/feature.tsx

```diff
@@ -200,4 +225,4 @@ class FeatureContainer extends React.Component {
   }
 }
 
-export default withConfig(FeatureContainer);
+export default withConfig(withOrganization(withProject(FeatureContainer)));
```

Comment:
Why wrap this, we are already pulling out the org and project from the context in the `FeatureContainer`?

Reply:
Ah, thanks. Meant to delete that and use the HoC